### DISCUSSION TITLE
budibase-fullcalendar Read out specific fields

### DISCUSSION THREAD

**Discussion by Sagestum** [Budibase/budibase#12219] [Help] [answered]
Hello all, 

Actually, I just wanted to create a little holiday calendar. Unfortunately, somehow budibase does not want to implement my query. 

If you click on an event in the calendar, it should jump to a modal page. for this I use a UniqID. 

When I click on an event, it also returns all the information as a json: 

`{
    "allDay": true,
    "title": "James Bond",
    "start": "2023-10-31",
    "end": "2023-11-04",
    "backgroundColor": "var(--spectrum-global-color-orange-700)",
    "borderColor": "var(--spectrum-global-color-orange-700)",
    "extendedProps": {
        "event": {
            "Title": "James Bond",
            "Von": "2023-10-31T11:00:00.000Z",
            "Bis": "2023-11-04T00:00:00.000Z",
            "Uniqid": "888",
            "_id": "%5B2%5D",
            "tableId": "datasource_plus_9b8e54d7dbe6486aa1f64528b5897af7__BRurlaub",
            "_rev": "rev"
        }
    }
}`

In my opinion, it should then only give me the uniqid with this query: 

`{{ Clicked Event.extendedProps.event.Uniqid }}`

but nothing comes back. 

Even if I simply 

`{{ Clicked Event.extendedProps }}`

there is no output. 

I suspect the problem is in front of the computer, can anyone give me some good advice?

Thanks
Chris

**Comment by ConorWebb96** [comment, marked as answer]
Hey @Sagestum,

I'm going to mark this as closed as you found a solution with Javascript. Any additional workarounds you find feel free to post them here. 

I believe the solution you came up with was:

```js
var uniqid = $("Clicked Event").extendedProps.event.Uniqid;
return(uniqid);
```

Hope this helps someone else using the same custom calendar plugin you used.
